### PR TITLE
Write obsoletes tag to primary

### LINKED ISSUE #503: Implement obsoletes tag
Let's add `obsoletes` tag to primary xml with corresponding dependencies names and versions.

### PR DESCRIPTION
**Pull request by olenagerasimova**
CLoses #503 
Added obsoletes tag to primary xml and corresponding tests. Also, exclude `/boot` folder from files list.

### CODE REVIEW

**Comment by genryxy** [comment]
@olenagerasimova build is failing now

File: src/main/java/com/artipie/rpm/meta/XmlEventPrimary.java [hunk 1]
```diff
@@ -118,9 +119,11 @@ public void add(final XMLEventWriter writer, final Package.Meta meta) throws IOE
             );
             XmlEventPrimary.addProvides(writer, tags);
             XmlEventPrimary.addRequires(writer, tags);
+            XmlEventPrimary.addObsoletes(writer, tags);
             // @checkstyle BooleanExpressionComplexityCheck (10 lines)
             new Files(
                 name -> name.startsWith("/var/")
+                    || name.startsWith("/boot") || name.startsWith("/boot/")
```
Comment (genryxy) [inline src/main/java/com/artipie/rpm/meta/XmlEventPrimary.java:126]: the first condition includes the second one

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one comment

**Comment by olenagerasimova** [comment]
@genryxy thanks, fixed

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**